mono/metadata/object.c

```diff
@@ -6928,6 +6931,7 @@ mono_value_box_handle (MonoDomain *domain, MonoClass *klass, gpointer value, Mon
 	error_init (error);
 
 	g_assert (m_class_is_valuetype (klass));
+	g_assert (value != NULL);
```

Comment:
@lambdageek  This assert would be nice to have on 2019-02. Maybe you have already proved me wrong but I still suspect `[MemberData]` getting into trouble using reflection to transform C# tuples to argument lists.

Reply:
@alexischr Can you point us at an example of the tuples transformation? Sounds like another interesting test case.

Reply:
@lambdageek Unfortunately that was back in the summer, I don't remember any details. Digging into the source brought it back (it was params object[]): https://github.com/xunit/xunit/commit/ff6ecaaf4df79f4266e7d2ab24493e49a5133031#diff-c6cbc959191da11fa205494c2731efe9L27 . Some of those methods look great for testing :)